modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/change/tracking/spi/reference/DLFileEntryTableReferenceDefinition.java

```diff
@@ -78,6 +79,9 @@ public void defineChildTableReferences(
 					ClassNameTable.INSTANCE.value.eq(FileEntry.class.getName())
 				)
 			)
+		).singleColumnReference(
+			DLFileEntryTable.INSTANCE.fileEntryId,
+			FriendlyURLEntryTable.INSTANCE.classPK
```

Comment:
Shouldn't this be a `classNameReference`? See `BlogsEntryTableReferenceDefinition`:
```
	@Override
	public void defineChildTableReferences(
		ChildTableReferenceInfoBuilder<BlogsEntryTable>
			childTableReferenceInfoBuilder) {

		childTableReferenceInfoBuilder.assetEntryReference(
			BlogsEntryTable.INSTANCE.entryId, BlogsEntry.class
		).classNameReference(
			BlogsEntryTable.INSTANCE.entryId,
			FriendlyURLEntryTable.INSTANCE.classPK, BlogsEntry.class
		).resourcePermissionReference(
			BlogsEntryTable.INSTANCE.entryId, BlogsEntry.class
		);
	}
```

Reply:
Hi @adolfopa 

The problem here is that we should put to this works is
```
		.classNameReference(
			DLFileEntryTable.INSTANCE.fileEntryId,
			FriendlyURLEntryTable.INSTANCE.classPK, FileEntry.class
		)
```
but this is an compilation error
![image](https://user-images.githubusercontent.com/47524751/177308360-39fbce40-6ca3-443f-8463-dc36ddeea4b1.png)
because the base model should be :

```
	.classNameReference(
			DLFileEntryTable.INSTANCE.fileEntryId,
			FriendlyURLEntryTable.INSTANCE.classPK, DLFileEntry.class
		)
```
And we are adding FriendlyURLEntry with the classnameId of FileEntry so the tests fails because never finds them.